config.sh

```diff
@@ -1,36 +1,26 @@
 vimrc_path="$HOME/.dotfiles/vimrc"
```

Comment:
NOTE: this file is getting bigger as we are adding stuff. Can you think of an intelligent way not to repeat yourself and clean up the code to make it more efficient?